collision_avoidance_node:
  ros__parameters:
    safe_distance: 0.6
    slow_distance: 1.2
    base_speed: 0.4
    turn_speed: 0.8
